msteams-platform/bots/how-to/bot-configuration-experience.md

```diff
@@ -54,95 +65,673 @@ You must configure the `fetchTask` property under `bots.configuration` object in
   ],
 ```
 
-For more information, see [public developer preview app manifest schema](../../resources/schema/manifest-schema-dev-preview.md#botsconfiguration).
+For more information, see [app manifest schema](../../resources/schema/manifest-schema.md#botsconfiguration).
 
 ### Configure your bot
 
-When a user installs the bot in a team or group chat scope, the `fetchTask` property in the app manifest file initiates `config/fetch` defined in the `teamsBot.js` file. The bot responds with an Adaptive Card and the user provides relevant information in the Adaptive Card and selects **Submit**. After the user selects **Submit**, a `config/submit` is returned to the bot and the bot configuration is complete.
+When a user installs the bot in a team or group chat scope, the `fetchTask` property in the app manifest file initiates `config/fetch` or `config/submit`. The bot responds with an Adaptive Card, when the user provides relevant information in the Adaptive Card and selects Submit, `config/submit` or `config/continue` is returned to the bot and the bot configuration is complete.
+
+You can use the following methods to enable configuration settings for a bot:
+
+1. [Non-Teams method](#non-teams-method).
```

Comment:
Remove this entire "Non-Teams method" section. Why do we need it?
Can't we just say update to BF SDK version xxx, and use the HandleTeamsConfig set of handlers?

Reply:
Not sure if the other API include onInvokeActivity or not, theoretically the develop can use both onInvoke or SDK method, but if the other API only mention the SDK method not the onInvoke one, I'm okay to remove this part.